### PR TITLE
Add @node in typeDefs

### PR DESCRIPTION
**Pull request by MacondoExpress**
Add `@node` in the documentation for the types that are mapped to a Neo4j node as planned for future release.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-170.surge.sh

Updated pages:
- [Autogeneration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/autogeneration)
- [Custom directives](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-directives)
- [Custom logic](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-logic)
- [Database mapping](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/database-mapping)
- [Indexes and constraints](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/indexes-and-constraints)
- [Field configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/field-configuration)
- [Global configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/global-configuration)
- [Type configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/type-configuration)
- [Driver configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/driver-configuration)

File: modules/ROOT/pages/subscriptions/events.adoc [hunk 1]
```diff
@@ -747,7 +747,7 @@ subscription {
                 }
            }
         }
-    }
+    }As an example, consider these type definitions:
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/subscriptions/events.adoc]: this is inside a code listing - intended?

File: modules/ROOT/pages/directives/database-mapping.adoc [hunk 2]
```diff
@@ -183,7 +183,21 @@ The following query yields a different Cypher query depending on the user JWT:
   users {
     name
   }
-}ref::/directives/database-mapping.adoc#_declarerelationship[
+}
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/directives/database-mapping.adoc]: odd, this should be fixed in 5.x now

File: modules/ROOT/pages/subscriptions/events.adoc (hunk 1 above)
Comment (MacondoExpress) [reply modules/ROOT/pages/subscriptions/events.adoc]: Nice catch!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-170.surge.sh

Updated pages:
- [Autogeneration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/autogeneration)
- [Custom directives](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-directives)
- [Custom logic](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/custom-logic)
- [Database mapping](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/database-mapping)
- [Indexes and constraints](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/indexes-and-constraints)
- [Field configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/field-configuration)
- [Global configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/global-configuration)
- [Type configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/directives/schema-configuration/type-configuration)
- [Driver configuration](https://neo4j-docs-graphql-170.surge.sh/graphql/5/driver-configuration)

File: modules/ROOT/pages/directives/database-mapping.adoc (hunk 2 above)
Comment (MacondoExpress) [reply modules/ROOT/pages/directives/database-mapping.adoc]: The branch was not updated, after the update this is resolved!

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Comment by MacondoExpress** [comment]
@rsill-neo4j  I've added a last commit with some rephrasing over the `@node` directive. I also moved the `@node` description to the top of the page as it will become the most fundamental block of the future version of the library. Can you have an extra check before merging it?

**Comment by MacondoExpress** [comment]
@rsill-neo4j This is closed as we plan to add this to version 6.

**Pull request closed without merging**